test/integration/targets/sqitch/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+- block:
+  - name: Create dir for resources
+    file: path=/tmp/sqitch_test state=directory mode=0755
```

Comment:
Use the `output_dir` variable to determine where temporary files should go instead of hard-coding to `/tmp/`.

Reply:
I don't know why {{ output_dir }} works in test.yml, but not in setup.yml. You can see what I mean here https://app.shippable.com/github/ansible/ansible/runs/84001/37/console
```
01:32 TASK [sqitch : Copy resources to tmp dir] **************************************
01:35 changed: [testhost] => {"changed": true, "dest": "/root/ansible_testing/", "src": "/root/ansible/test/integration/targets/sqitch/files/"}
```
but below
```
"cwd": "~/ansible_testing"
```